### PR TITLE
Data Changes

### PR DESCRIPTION
**Pull request by a3957273**

### CODE REVIEW

File: .vscode/settings.json
Comment (a3957273) [inline .vscode/settings.json:1]: Don't keep

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
+    "printWidth": 120,
+    "semi": false,
+    "singleQuote": true,
+    "jsxSingleQuote": true
+  },
   "editor.formatOnSave": true,
   "editor.codeActionsOnSave": {
     "source.fixAll.eslint": true
   },
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
```
Comment (ARADDCC012) [inline .vscode/settings.json:14]: No harm in keeping this line, but the rest can go

File: backend/src/scripts/example_schemas/minimal_upload_schema_beta.json
Comment (ARADDCC012) [inline backend/src/scripts/example_schemas/minimal_upload_schema_beta.json:1]: Don't keep, but make a note of any parts of the schema that should use `richTextInput`

File: frontend/public/css/table.css
Comment (ARADDCC012) [inline frontend/public/css/table.css:1]: Keep this

File: frontend/src/Form/beta/ModelCardForm.tsx
Comment (ARADDCC012) [inline frontend/src/Form/beta/ModelCardForm.tsx:1]: Keep this

File: frontend/src/MuiForms/CustomTextInput.tsx
Comment (ARADDCC012) [inline frontend/src/MuiForms/CustomTextInput.tsx:1]: Keep this

File: frontend/src/MuiForms/RichTextInput.tsx
Comment (ARADDCC012) [inline frontend/src/MuiForms/RichTextInput.tsx:1]: Keep this

File: frontend/src/common/RichTextEditor.tsx
Comment (ARADDCC012) [inline frontend/src/common/RichTextEditor.tsx:1]: Keep this

File: frontend/src/common/MarkdownRenderer.tsx
Comment (ARADDCC012) [inline frontend/src/common/MarkdownRenderer.tsx:1]: Keep this

File: backend/.prettierignore
Comment (a3957273) [inline backend/.prettierignore:1]: Keep

File: backend/src/models/v2/Release.ts
Comment (a3957273) [inline backend/src/models/v2/Release.ts:1]: Don't keep

File: frontend/actions/model.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ interface ModelSearchResult {
   tags: Array<string>
 }
 
-export function useListModels(type: ListModelType, filter?: string) {
+export function useListModels(type: ListModelType, search?: string) {
```
Comment (JR40159) [inline frontend/actions/model.ts:15]: Model Search Fix: use correct param

File: backend/src/routes/v1/registryAuth.ts
Comment (a3957273) [inline backend/src/routes/v1/registryAuth.ts:1]: Don't keep

File: backend/src/routes/v2/model/getModelsSearch.ts
Comment (a3957273) [inline backend/src/routes/v2/model/getModelsSearch.ts:1]: Don't keep

File: frontend/actions/model.ts [hunk 3]
```diff
@@ -21,7 +21,7 @@ export function useListModels(type: ListModelType, filter?: string) {
   >(
     `/api/v2/models/search?${qs.stringify({
       type,
-      filter,
+      search,
```
Comment (JR40159) [inline frontend/actions/model.ts:24]: Model Search Fix: use correct param

File: backend/src/routes/v2/release/postRelease.ts
Comment (a3957273) [inline backend/src/routes/v2/release/postRelease.ts:1]: Don't keep

File: backend/src/routes/v2/review/getReview.ts
Comment (a3957273) [inline backend/src/routes/v2/review/getReview.ts:1]: Don't keep

File: backend/src/scripts/example_schemas/minimal_upload_schema_beta.json
Comment (a3957273) [inline backend/src/scripts/example_schemas/minimal_upload_schema_beta.json:1]: Don't keep

File: frontend/package.json
Comment (a3957273) [inline frontend/package.json:1]: Keep

File: frontend/pages/_document.tsx
Comment (a3957273) [inline frontend/pages/_document.tsx:1]: Sorta keep?

File: frontend/pages/beta/index.tsx [hunk 4]
```diff
@@ -156,17 +157,17 @@ export default function ExploreModels() {
                         variant='h5'
                         sx={{ fontWeight: '500', textDecoration: 'none', color: theme.palette.primary.main }}
                       >
-                        {model.name}
+                        Example Team / {model.name}
```
Comment (a3957273) [inline frontend/pages/beta/index.tsx:160]: Don't keep

File: frontend/public/css/table.css [hunk 5]
```diff
@@ -1,5 +1,4 @@
 table {
-  margin: auto;
```
Comment (a3957273) [inline frontend/public/css/table.css:2]: Keep

File: frontend/src/common/MarkdownRenderer.tsx
Comment (a3957273) [inline frontend/src/common/MarkdownRenderer.tsx:1]: Keep

File: frontend/src/model/beta/overview/FormEditPage.tsx [hunk 6]
```diff
@@ -68,7 +68,7 @@ export default function FormEditPage({ model }: FormEditPageProps) {
   return (
     <>
       {(isSchemaLoading || isUiConfigLoading) && <Loading />}
-      <Box sx={{ py: 1 }}>
+      <Box sx={{ py: 1, px: 2 }}>
```
Comment (a3957273) [inline frontend/src/model/beta/overview/FormEditPage.tsx:71]: Don't keep

File: frontend/src/model/beta/releases/DraftNewReleaseDialog.tsx
Comment (a3957273) [inline frontend/src/model/beta/releases/DraftNewReleaseDialog.tsx:1]: Don't keep

File: frontend/src/model/beta/releases/ModelReleaseDisplay.tsx [hunk 7]
```diff
@@ -105,20 +103,19 @@ export default function ModelReleaseDisplay({
                 </Typography>
                 <Typography variant='caption'>{release.createdBy}</Typography>
               </Stack>
-              <Typography variant='body1'>{release.notes}</Typography>
-              <Divider />
+              <Markdown>{release.notes}</Markdown>
```
Comment (a3957273) [inline frontend/src/model/beta/releases/ModelReleaseDisplay.tsx:106]: Keep

File: frontend/src/model/beta/releases/ModelReleaseDisplay.tsx [hunk 8]
```diff
@@ -105,20 +103,19 @@ export default function ModelReleaseDisplay({
...
+              {(release.files.length > 0 || release.images.length > 0) && <Divider />}
               <Stack spacing={0}>
                 {release.files.map((file) => (
                   <Stack
-                    key={file}
+                    key={file.name}
```
Comment (a3957273) [inline frontend/src/model/beta/releases/ModelReleaseDisplay.tsx:111]: Keep

File: frontend/src/model/beta/releases/ModelReleaseDisplay.tsx [hunk 9]
```diff
@@ -105,20 +103,19 @@ export default function ModelReleaseDisplay({
...
                     direction={{ sm: 'row', xs: 'column' }}
                     justifyContent='space-between'
                     alignItems='center'
                     spacing={2}
                   >
-                    <Link href='/beta'>{file}</Link>
-                    {/* TODO - Add file size here */}
-                    {/* <Typography variant='caption'>123GB</Typography> */}
+                    <Link href='/beta'>{file.name}</Link>
```
Comment (a3957273) [inline frontend/src/model/beta/releases/ModelReleaseDisplay.tsx:117]: Keep

File: frontend/src/model/beta/settings/AccessRequestSettings.tsx [hunk 10]
```diff
@@ -11,7 +11,16 @@ export default function AccessRequestSettings() {
       </Typography>
       <Stack direction='row' alignItems='center'>
         <Checkbox onChange={(event) => setAllowUngoverned(event.target.checked)} value={allowUngoverned} size='small' />
-        <Typography>Allow users to make ungoverned access requests</Typography>
+        <Typography>
```
Comment (a3957273) [inline frontend/src/model/beta/settings/AccessRequestSettings.tsx:14]: Don't keep?

File: frontend/types/types.ts
Comment (a3957273) [inline frontend/types/types.ts:1]: Don't keep

File: frontend/types/types.ts [hunk 11]
```diff
@@ -398,16 +398,19 @@ export interface SchemaInterface {
 
 export const SchemaKind = {
   Model: 'model',
-  Deployment: 'deployment',
+  Deployment: 'accessRequest',
```
Comment (a3957273) [inline frontend/types/types.ts:401]: Keep

File: frontend/utils/fetcher.ts [hunk 12]
```diff
@@ -38,12 +38,24 @@ export const fetcher = async (input: RequestInfo, init: RequestInit) => {
     if (res.status === 401) {
       redirectToLoginPage()
     }
-    const error: ErrorInfo = {
-      ...new Error('An error occurred while fetching the data.'),
-      info: await res.json(),
-      status: res.status,
+
```
Comment (a3957273) [inline frontend/utils/fetcher.ts:41]: Keep

File: lib/python-beta/src/bailo/bailo.py
Comment (a3957273) [inline lib/python-beta/src/bailo/bailo.py:1]: Keep

File: frontend/src/common/MarkdownRenderer.tsx
Comment (ARADDCC012) [reply frontend/src/common/MarkdownRenderer.tsx:1]: I'll include this in my branch as it's needed for the markdown widget

**Pull request closed without merging**